### PR TITLE
possible to import now

### PR DESCRIPTION
**Pull request by pwalawko**

### CODE REVIEW

File: arithmetic.py [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
-        r = int(sys.argv[2])
+        a1 = int(a1)
+        n = int(n)
+        r = int(r)
     except ValueError:
         print('Incorrect parameter!')
+    else:
+        print(a1 + (n - 1) * r)
+
+if __name__ == '__main__':
```
Comment (kurazu) [inline arithmetic.py]: :+1:

**Comment by kurazu** [comment]
This is what I want to be able to do:
```
>>> import arithmetic
>>> x = arithmetic.counting(1, 2, 3)
>>> y = arithmetic.counting(3, 4, 5)
>>> z = arithmetic.counting(6, 7, 8)
>>> print('x={x} y={y} z={z}'.format(x=x, y=y, z=z))
x=6 y=12 z=21
```
*(the values are fake)*

So I want to be able to use just the **calculations** from the outside.
The calculations should **return** a result, not *print* it.
*Parsing* parameters is **not** part of calculations.
*Yes*, the calculations themselves will be roughly a one-liner.

*Why* is this important?
The code needs to have a clear separation of *UI interactions* and *business logic*. Passing arguments from the command line is our crude UI in this program. Handling this UI: extracting parameters, validating them and informing the user about them being incorrect is part of UI interactions. *So* is informing the user about the result of his computations (printing it on the screen).
The *business logic* we have is just calculating the n-th element of an arithmetic sequence. Nothing more, nothing less. Taking the parameters (assumed to be of correct types), doing math and **returning** the result to whomever is calling a function.

Last note on ```'__main__'```:
It's a good custom to put all that code in a function called ```def main():``` and only have:
```
if __name__ == '__main__':
    main()
```
*Why* is this important?
If you are putting your code like this:
```
if __name__ == '__main__':
    a = 3
    ...
```
then you are *leaking* a variable called ```a``` into the *module scope*. Somebody could do:
```
import arithmetic
print(arithmetic.a)
```
and get the value of that (internal) variable, or worse
```
import arithmetic
arithmetic.a = 'ninja'
```
modify it.

**Comment by pwalawko** [comment]
fixed

**Comment by kurazu** [comment]
Looks good now :+1:

**Pull request merged**